### PR TITLE
rd

### PR DESCRIPTION
**Pull request by maxwellbenton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,122 @@
...
+we've already met. It looks like this:
+
+```ruby
+# Position 1                # Position 2                # Position 3
+boolean_conditional_value ? "thing_to_return_if_true" : "thing_to_return_if_false"
+```
+
+If the expression (_constant expression_ with a Boolean value, _assignment
+expression_,  etc.) is a `true` value, then the return value of the expression
+is "thing_to_return_if_false". If it is `false`, "thing_to_return_if_false` is
```
Comment (maxwellbenton) [inline README.md]: should be " not `

File: README.md [hunk 2]
```diff
@@ -1,16 +1,122 @@
...
+This logic makes `garment` `"galoshes"` again. Try swapping `>` for `<`, is
+that what you expected?
+
+You've now met two _comparison operators_. Comparison operators return `true`
+or `false`. We'll meet the whole family of them in the next lesson. You can
+always swap a "literal constant value" for "an expression that returns a value"
+in programming. Thus since comparison operations return `true` or `false`, we
+can use one of those expressions in Position 1 of our ternary statement.
+
+Even more awamzing, we can swap out the simple `String`s we've used in
```
Comment (maxwellbenton) [inline README.md]: a _wam_ zing

File: README.md [hunk 3]
```diff
@@ -1,16 +1,122 @@
...
+Positions 2 and 3 of our ternary with expressions as well.
+
+```ruby
+top = 2
+bottom = 1
+problem_count = 99
+lucky_number = (top > bottom) ? ( 3 + 1 ) : ( problem_count / 3 ))
```
Comment (maxwellbenton) [inline README.md]: extra ) at the end

File: README.md (hunk 2 above)
Comment (sgharms) [reply README.md]: Probably a feature not a bug. Awamzing sounds cool.

**Pull request merged**